### DISCUSSION TITLE
[HELP] Can not expose `state_reason` from Github Issues

### DISCUSSION THREAD

**Discussion by bracesproul** [octokit/octokit.js#2345] [Q&A] [answered]
I'm trying to get the `state_reason` response from the get github issues rest route and I keep getting errors saying `Property 'state_reason' does not exist on type {...<type here>}`. The type is derived from:
```ts
Octokit['rest']['issues']['get']
```
I also can't find any mention of `state_reason` in any of the types files in the rest, core, and octokit repos. All this is making me think I can't access the `state_reason` property/I can only access it through another rest route that I am currently unaware of. Does anyone have any insight on this?

Github API reference: https://docs.github.com/en/rest/issues/issues#get-an-issue

**Comment by wolfy1339** [comment, marked as answer]
Can you share what versions of Octokit you are using? Is this an error from Node, or from TypeScript?

I can't reproduce the issue. [Playground Link](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgeQMYwga2PAvnAMyghDgCIABCdLHAeigFMBnGMgKHdQgDtW5qGbPAC8cHowDuKGsIAUASgDcnbn3hIAJgEMY2uPjHbJ2nANk4AdE1aXgzZgFcWlgOaMYchOzi+-vpkgALnJtHgBPMgAaH38-CEkJKBCydAAPa0ZISwSk6Ni48nsnRgB9HkcQACNGKDIQgEYCv1xlTh09S1ZdMqZtZl4lIA)

The types for the rest endpoints lives in `@octokit/openapi-types` and `@octokit/types` packages

**Reply by bracesproul** [reply]
Thank you for that, I found it inside this file: https://raw.githubusercontent.com/octokit/openapi-types.ts/main/packages/openapi-types/types.d.ts

I'm guessing the Octokit version I'm running is older than the version where this field was added because the actual Github API only added that field earlier this year.

I'm running `octokit: ^1.7.1`

**Reply by wolfy1339** [reply]
Yeah, that would most likely be your issue then.
The latest version of `octokit` is `v2.0.10`
